README.md

```diff
@@ -1,6 +1,6 @@
 # Thorn [![Build Status](https://travis-ci.org/sugarcrm/thorn.svg?branch=master)](https://travis-ci.org/sugarcrm/thorn) [![Coverage Status](https://coveralls.io/repos/github/sugarcrm/thorn/badge.svg?branch=master)](https://coveralls.io/github/sugarcrm/thorn?branch=master)
 
-Thorn is composed of a set of helper abstractions built to ease the process of setting up a SugarCRM instance's REST API testing environment and interacting with it.
+Thorn is composed of a set of helper abstractions built to ease the process of setting up a [SugarCRM](https://www.sugarcrm.com) instance's REST API testing environment and interacting with it.
```

Comment:
My personal rule for live sites is: never use plain HTTP unless HTTPS is unsupported or broken (eg. due to mixed content).